pkg/binder/framework/plugins/interpodaffinity/interpodaffinity.go

```diff
@@ -83,3 +93,29 @@ func New(_ runtime.Object, handle handle.BinderFrameworkHandle) (framework.Plugi
 		frameworkHandle: handle,
 	}, nil
 }
+
+func (pl *InterPodAffinity) updateStateByVictims(state *utils.PreFilterState, handle handle.BinderFrameworkHandle, victimsGroupByNode map[string]map[types.UID]*v1.Pod, targetPod *v1.Pod) error {
```

Comment:
It doesn't need to be a function of pl. Or remove the parameter `handle`.

Reply:
I chose to remove the parameter `handle`. Because similar functions `AddPod` and `RemovePod` in shceduler are also functions of plugin itself.